packages/epetra/cmake/CheckHaveLapackGSSVD3.cmake

```diff
@@ -0,0 +1,37 @@
+# LAPACK 3.6.0 deprecates dggsvd and sggsvd (Issue #480)
+# The new versions of dggsvd and sggsvd are called dggsvd3 and sggsvd3
+#
+# dggsvd (JOBU, JOBV, JOBQ, M, N, P, K, L, A, LDA, B, LDB, ALPHA, BETA, U, LDU, V, LDV, Q, LDQ, WORK, IWORK, INFO)
+# sggsvd (JOBU, JOBV, JOBQ, M, N, P, K, L, A, LDA, B, LDB, ALPHA, BETA, U, LDU, V, LDV, Q, LDQ, WORK, IWORK, INFO)
+#
+# dggsvd3 (JOBU, JOBV, JOBQ, M, N, P, K, L, A, LDA, B, LDB, ALPHA, BETA, U, LDU, V, LDV, Q, LDQ, WORK, LWORK, IWORK, INFO)
+# sggsvd3 (JOBU, JOBV, JOBQ, M, N, P, K, L, A, LDA, B, LDB, ALPHA, BETA, U, LDU, V, LDV, Q, LDQ, WORK, LWORK, IWORK, INFO)
+#
+# The new parameter is LWORK
+#
+#   Information is at:
+#   http://www.netlib.org/lapack/explore-html/d6/db3/dggsvd3_8f_a4a187519e5c71da3b3f67c85e9baf0f2.html#a4a187519e5c71da3b3f67c85e9baf0f2
+
+INCLUDE(CheckFunctionExists)
+
+FUNCTION(CHECK_HAVE_LAPACK_GSSVD3 VARNAME)
+
+  SET(CMAKE_REQUIRED_LIBRARIES ${TPL_LAPACK_LIBRARIES})
+
+  CHECK_FUNCTION_EXISTS( dggsvd3   HAVE_dggsvd3 )
```

Comment:
Lines 21-24 limit the set of possible Fortran manglings.  Why not just use the existing CMake FortranCInterface (de)mangling functions?